attributes/default.rb

```diff
@@ -14,3 +10,19 @@
 default["collectd"]["python_plugins"]     = Mash.new
 default["collectd"]["graphite_role"]      = "graphite"
 default["collectd"]["graphite_ipaddress"] = ""
+
+default["collectd"]["source"]["version"]            = "5.4.1"
```

Comment:
Because all of these variables are being renamed, could you also update the references in the `README`?